src/connect/index.js

```diff
@@ -1,17 +1,17 @@
 "use strict";
 
 var async = require("async");
-var ethereumConnector = require("ethereumjs-connect");
```

Comment:
It looks like ethereumjs-connect is still used in the augur-node and augur repos. I assume the plan to remove it from those as well once this is merged in?